source/partials/next_release/_release.md.erb

```diff
@@ -35,21 +49,23 @@
 
 - <%= link_to_commit 'f2ac6b182c03413e8c6cdbae26c47fe944a11897', 'Removed support for text based API support log.', 2975 %>
 
+- <%= link_to_commit 'da61c2bd7ee7e95dcf35d67867d5336efee76c83', 'Remove old agents page', 3015 %>
 
 #### Deprecations
 
 Features that become superseded or irrelevant become *deprecated*, and are scheduled to be removed in some timeframe.
 
 The following features have been deprecated, along with the timeframe when they will be removed.
 
-* We shall be stopping support for git versions older than 1.9 in three months time (Feb 2017). Users are recommended to upgrade git on the GoCD server and agents (if git is used as a material). We are aware that the linux distributions have an older version of git installed by default. Please refer to [this](https://www.softwarecollections.org/en/scls/rhscl/git19/) and/or [this](http://askubuntu.com/questions/571549/git-1-7-9-5-upgrade-to-current-release-of-git-2-x-on-ubuntu-12-04/571633#571633) to upgrade git.
+* As of release 16.12, git versions older than 1.9 are deprecated. Support for git versions older than 1.9 will be stopped in 17.2 (scheduled to be released in February 2017). Users are recommended to upgrade git on the GoCD server and agents (if git is used as a material). We are aware that the linux distributions have an older version of git installed by default. Please refer to [this](https://www.softwarecollections.org/en/scls/rhscl/git19/) and/or [this](http://askubuntu.com/questions/571549/git-1-7-9-5-upgrade-to-current-release-of-git-2-x-on-ubuntu-12-04/571633#571633) to upgrade git.
 * As of release 16.12, version 1 of the <%= link_to_api 'get-all-plugin-info', 'Plugin Info API' %> is deprecated. This version of the API will be removed in 17.2 release (scheduled to be released in Feb 2017). Version 2 of the API is available, and users are encouraged to use it.
+* As of release 17.1, version 2 of the <%= link_to_api 'template-config', 'Templates API' %> is deprecated. This version of the API will be removed in 17.3 release (scheduled to be released in March 2017). Version 3 of the API is available, and users are encouraged to use it.
```

Comment:
we give a 3 month deprecation period for APIs. Version 2 will be removed in 17.4 (Scheduled for April 2017)